db/migrate/20160719042908_add_unicode_to_piece.rb

```diff
@@ -0,0 +1,5 @@
+class AddUnicodeToPiece < ActiveRecord::Migration
```

Comment:
Since you defined the `unicode_symbol` method on the pieces classes, you don't need to add it to the DB here.